packages/react-form-renderer/src/index.js

```diff
@@ -13,6 +13,7 @@ export { default as RendererContext } from './components/renderer-context';
 export { default as FormSpy } from './components/form-spy';
 export { default as Form } from './components/form';
 export { default as FieldArray } from './components/field-array';
+export { default as useFieldProviderApi } from './hooks/use-field-provider-api';
```

Comment:
What about the name? I think it's too long and for other people who doesn't know implementation, it can be unnecessary specific. (_What is a provider?_)

I think `useFieldApi` (and it will be consistent with `useFormApi`) sounds better (as it is basically just wrapped useField from FF)